### PR TITLE
Value Pool Fuzzer, with current status, only generates full recombinations

### PR DESCRIPTION
**Pull request by jonathanheitzmann**

### CODE REVIEW

File: custom_components/test/fuzzing/fuzzer_utils/ValuePool.py [hunk 1]
```diff
@@ -57,10 +50,11 @@ def __init__(self) -> None:
             257.0,
             sys.maxsize * 0.5,
             sys.maxsize * sys.maxsize * 0.5,
-        ]
+        ] + self.__INT_POOL
```
Comment (ThorbenCarl) [inline custom_components/test/fuzzing/fuzzer_utils/ValuePool.py:53]: Does it make sense that the `__FLOAT_POOL` also have `int` values?

File: custom_components/test/fuzzing/fuzzer_utils/ValuePool.py [hunk 2]
```diff
@@ -71,18 +65,23 @@ def __init__(self) -> None:
         ]
 
         # set values for _BOOL_POOL
-        self._BOOL_POOL = [True, False]
+        self.__BOOL_POOL = [
+            None,
```
Comment (ThorbenCarl) [inline custom_components/test/fuzzing/fuzzer_utils/ValuePool.py:69]: Does it make sense that the `__BOOL_POOL` also has a `None` value?

File: custom_components/test/fuzzing/fuzzer_utils/ValuePool.py [hunk 3]
```diff
@@ -57,10 +50,11 @@ def __init__(self) -> None:
             257.0,
             sys.maxsize * 0.5,
             sys.maxsize * sys.maxsize * 0.5,
-        ]
+        ] + self.__INT_POOL
 
         # set values for _STRING_POOL
-        self._STRING_POOL = [
+        self.__STRING_POOL = [
+            None,
```
Comment (ThorbenCarl) [inline custom_components/test/fuzzing/fuzzer_utils/ValuePool.py:57]: Does it make sense that the `__STRING_POOL` also has a `None` value?

**Review by ThorbenCarl — COMMENTED** [review]
@jonathanheitzmann please check if I have not broken anything by removing the merge conflicts.

**Review by JKortmann — COMMENTED** [review]
Same points as @ThorbenCarl

**Review by ThorbenCarl — APPROVED** [review]

**Pull request merged**